dev-python/docstring-to-markdown/docstring-to-markdown-0.9.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="On the fly conversion of Python docstrings to markdown"
+HOMEPAGE="https://github.com/krassowski/docstring-to-markdown"
+# PyPI tarball doesn't include tests
+SRC_URI="https://github.com/krassowski/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	test? (
+		dev-python/pytest-cov[${PYTHON_USEDEP}]
```

Comment:
`pytest-cov` is an undesirable test dependency according to: https://dev.gentoo.org/~mgorny/python-guide/distutils.html#enabling-tests

Usually it is possible to remove this dependency by editing the pytest.ini or setup.cfg file

Reply:
Thanks, fixed with some `sed`.